module/observationSubmissions/helper.js

```diff
@@ -1000,13 +1009,19 @@ module.exports = class ObservationSubmissionsHelper {
           entityIds.push(submission.entityId);
         });
 
-        let entitiesData = await entitiesHelper.entityDocuments(
+        let entitiesDetails = await entityManagementService.entityDocuments(
           {
             _id: { $in: entityIds },
           },
           ['metaInformation.externalId', 'metaInformation.name'],
         );
 
+        if(!entitiesDetails.success){
+          throw new Error("Entity Management Service down.")
```

Comment:
@borkarsaish65 define this message as a response please

Reply:
done